make_html.py

```diff
@@ -0,0 +1,57 @@
+import os
+import yaml
+import shutil
+import jinja2
+
+
+def get_options():
+    import argparse
+    parser = argparse.ArgumentParser(
+        description="Get starcheck checks")
+    parser.set_defaults()
+    parser.add_argument("--infile",
+                        default="checks.yaml",
+                        help="file of checks (output from previous run of tool)")
+    parser.add_argument("--outdir",
+                        default="report_html")
+    args = parser.parse_args()
+    return args
+
+jinja_env = jinja2.Environment(
+    loader=jinja2.FileSystemLoader('templates'))
+jinja_env.line_comment_prefix = '##'
+jinja_env.line_statement_prefix = '#'
+
+
+opt = get_options()
+if not os.path.exists(opt.outdir):
+    os.makedirs(opt.outdir)
+shutil.copy("prism_ocadia.css", opt.outdir)
+shutil.copy("prism.js", opt.outdir)
+shutil.copy("sorttable.js", opt.outdir)
+
+checks = yaml.load(open(opt.infile).read())
+
+displaycols = ['id', 'link', 'filename', 'line_number',
```

Comment:
Regarding "missing", if we remove a warning from the code, do you want the "permalink" to it to still work?  Should I represent this by keeping the warning but setting the filename to null or some such?